code/datums/helper_datums/teleport.dm

```diff
@@ -120,16 +119,6 @@
 	else
 		if(teleatom.Move(destturf))
 			playSpecials(destturf,effectout,soundout)
-
-	if(isliving(teleatom))
-		var/mob/living/L = teleatom
-		if(L.buckled)
-			L.buckled.unbuckle_mob()
-		if(L.buckled_mob)
-			L.unbuckle_mob(force=1)
-
-	destarea.Entered(teleatom)

```

Comment:
Entered() is already called by forceMove() and Move() (line 118 and 121).
The unbuckling is already done in forceMove(), and Move() fails if we're buckled.